build/mediaelement-and-player.js

```diff
@@ -2407,6 +2406,81 @@ window.MediaElement = mejs.MediaElement;
 
 }(mejs.i18n.locale.strings));
 /*!
+ * This is a i18n.locale language object.
```

Comment:
#22 changed the gruntfile a bit without doing a build it looks like. I guess this is what that included in the build. @tennety might have more info.